### PR TITLE
shiftstack-ci: Add repository

### PR DESCRIPTION
**Pull request by pierreprinetti**
Part of [OSASINFRA-2502](https://issues.redhat.com/browse/OSASINFRA-2502)

### CODE REVIEW

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: shellcheck
+  commands: shellcheck ./clean-ci-resources.sh
+  container:
+    from: shellcheck
```
Comment (alvaroaleman) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:17]:
```suggestion
    from: src
```

**Comment by pierreprinetti** [comment]
Depends on https://github.com/openshift/release/pull/18999

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**